contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -144,6 +111,9 @@ contract LPOptimiserStrategy is DefaultAccessControl {
     }
 
     /// @notice Get the nearest tick multiple given a tick and tick spacing
+    /// @param newTick The tick to be rounded to the closest multiple of tickSpacing
+    /// @param tickSpacing The tick spacing of the vamm being used for this strategy
+    /// @return int24 The nearest tick multiple for newTick
     function nearestTickMultiple(int24 newTick, int24 tickSpacing) public pure returns (int24) {
```

Comment:
why does this function need to be public vs. internal?

Reply:
worth having unit tests for a couple of edge case scenarios for this function specifically if don't have already

Reply:
I have written UTs for this function. The function should ideally be internal but if it is internal we can't test it. So that makes things a bit difficult from a UT perspective. We could make it internal and then have something in the readme to say "Change the nearestTickMultiple function to public for tests to work"